### PR TITLE
dcu.robot: added keywords to boot to Windows or other OS

### PR DESCRIPTION
**Pull request by JanPrusinowski**

### CODE REVIEW

File: lib/options/dcu.robot [hunk 1]
```diff
@@ -147,3 +147,26 @@ Make Sure That Flash Locks Are Disabled
...
+
+    ${boot_entries}=    SSHLibrary.Execute Command    efibootmgr
+    @{lines}=    Split To Lines    ${boot_entries}
+    FOR    ${line}    IN    @{lines}
+        IF    '${os}' in '${line}'
+            ${os_boot_id}=    Set Variable    ${line}
+        END
+    END
+    ${os_boot_id}=    Split String    ${line}    separator=    max_split=1
+    ${id}=    Get Substring    ${os_boot_id}    4    7
```
Comment (mkopec) [inline lib/options/dcu.robot:170]: Don't we need to check if the OS boot entry actually exists here?

Comment (JanPrusinowski) [reply lib/options/dcu.robot]: added: 8e0c7704d287b909dc808fb3a87f8f7ab0fabe90

Comment (mkopec) [reply lib/options/dcu.robot]: Thanks.

**Review by mkopec — APPROVED** [review]

**Comment by mkopec** [comment]
@JanPrusinowski Please rebase and feel free to merge once rebased and CI passes

**Pull request merged**